### ISSUE TITLE
Refactor the different papers and films properties

### ISSUE THREAD

**Issue by daar** [mickadam/PinholeCalculator#11]
Investigate the best way to refactor the paper and film properties. Now these properties are hardcoded in Exposition.java. Investigate wether it makes sense to refactor them out in separate (inherited) classes, or to store the properties as polynomes in the database.
The latter makes it possible for users to create their own "custom" film or paper.

**Comment by mickadam** [comment]
I agree with that, the code needs much refactor, especially for paper properties.
